### DISCUSSION TITLE
Nightly build

### DISCUSSION THREAD

**Discussion by naturzukunft** [eclipse-rdf4j/rdf4j#4007] [General]
Is there something like a nightly build? i have trouble in my fork with the (hopefully (-: ) unchanged main branch. 
some deadlock test in rdf4j-sail-api are failing

**Comment by hmottestad** [comment]
https://rdf4j.org/download/#source-code-and-nightly-builds

**Comment by naturzukunft** [comment]
Ok. so there is no way to run a build on a prepared server for my fork/branch. I'm not conform with github. in gitlab i work with gitlab-ci. So there i can easily see, if the build is running in a stable env. On lokal machines there is also some "restlessness" ,-) I did now a `mvn verify` on a clone of the main repo. it fails later, but it fails ;-(
